torch/csrc/jit/ir/alias_analysis.cpp

```diff
@@ -1207,8 +1253,8 @@ void AliasDb::createValue(const Value* value) {
 void AliasDb::giveFreshAlias(
```

Comment:
`add_wildcard_to_contained_elems` is a micro-optimization for nodes like 

```
x = torch.tensor([2])
torch.cat([x, x, x, x])
```
Where we look if the list created is passed to a functional operator and then put x in the contained elements of the list instead of setting x to the wildcard set. Even if this parameter is `False`,  we still want to `pointUnionTypeElementToAllContainedTypes`. It maybe makes sense to rename the argument to something like `contained_elements_have_already_added` or something, i'm not sure.